packages/react-components/react-table/stories/DataGrid/KeyboardColumnResizing.stories.tsx

```diff
@@ -164,16 +167,47 @@ const columns: TableColumnDefinition<Item>[] = [
   }),
 ];
 
+const useStyles = makeStyles({
```

Comment:
Is there a reason that we recommend users to apply explicit styles instead of focusing on the cell that is being resized?

Reply:
Focus is on body while the resizing is happening, have you checked with our a11y experts whether this is a good experience?